src/components/fields/Colorscale.js

```diff
@@ -30,11 +30,20 @@ class Colorscale extends Component {
       ? fullValue.map(v => v[1])
       : null;
 
+    // removing label prop on colorscales, not enough space
+    const adjustedProps = {...this.props};
+    let fullWidth = false;
+    if (adjustedProps.label) {
+      fullWidth = true;
+      delete adjustedProps.label;
+    }
+
     return (
-      <Field {...this.props}>
+      <Field {...adjustedProps}>
```

Comment:
Could you just do `<Field {...this.props} label={false}>`?
I think if a props is defined twice, only the last one counts, so you can overwrite with an empty.

Reply:
that's nice, didn't think of that one thanks!